modules/mcuboot/CMakeLists.txt

```diff
@@ -264,6 +258,32 @@ if(CONFIG_BOOTLOADER_MCUBOOT)
         net_core_app_sign_target
         )
 
+      generate_dfu_zip(
+        TARGET mcuboot_sign_target
+        OUTPUT ${PROJECT_BINARY_DIR}/dfu_application.zip
+        BIN_FILES ${PROJECT_BINARY_DIR}/${app_core_binary_name} ${PROJECT_BINARY_DIR}/${net_core_binary_name}
+        TYPE application
+        SCRIPT_PARAMS
+        "${app_core_binary_name}load_address=$<TARGET_PROPERTY:partition_manager,PM_APP_ADDRESS>"
+        "${net_core_binary_name}load_address=missing"
```

Comment:
```suggestion
        "${net_core_binary_name}load_address=$<TARGET_PROPERTY:partition_manager,CPUNET_PM_APP_ADDRESS>"

Reply:
So the question here do we want the load address to be where the image ends up or the address it should be loaded too.

In general I would assume this argument will be ignored and just passed as SMP upload command with `image_slot` being the one which decides where the image is put.

Reply:
To be honest I don't think we need it at all. I added it in the draft as it was in json already and I don't think it was ever used anyway.